### DISCUSSION TITLE
How can I make multiple docs in my documentation

### DISCUSSION THREAD

**Discussion by tiluckdave** [shuding/nextra#343] [Q&A]
What I want is
```
pages
    - docs
        - index.mdx
    - docsv2
        - index.mdx
```
And I want 
`/docs` as a separate webpage
and `/docsv2` as a separate webpage

**Comment by tiluckdave** [comment]
Please help!

**Reply by ansh-devs** [reply]
Did u figured it out

**Reply by tahamaddam** [reply]
Looking forward to the solution.  

To make this clear, here is an example of the implementation in Vitepress, where they have one section(docs) for "[Guides](https://vitepress.dev/guide/what-is-vitepress)" and another section(docs) for "[Reference](https://vitepress.dev/reference/site-config)"

**Comment by ansi2u** [comment]
I think you can define docs &  docsv2 as separate folders under the pages folder. 
This will make docs accessible as `/docs` & `docsv2`
